haystack/utils/export_utils.py

```diff
@@ -16,58 +12,97 @@
 logger = logging.getLogger(__name__)
 
 
+FIELDS_TO_KEEP_BY_LEVEL = {
+    "minimum": ["answer", "context"],
+    "medium": ["answer", "context", "score"]
+}
```

Comment:
As this constant is only used in `print_answers` I would probably move it within the scope of the method.